### PR TITLE
Add cluster config for kind create cluster command

### PR DESCRIPTION
**Pull request by ChrsMark**
## What does this PR do?
This PR changes how we start kind cluster for k8s tests so as to be able to expose kube-proxy's metrics.

Related to https://github.com/elastic/integrations/pull/886, https://github.com/elastic/elastic-package/pull/322, https://github.com/elastic/elastic-package/tree/master/internal/install

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -185,7 +185,7 @@ def withKubernetes(Closure body) {
           chmod +x ${HOME}/bin/kubectl
           kubectl version --client
           ''') }
-        sh(label: "Create kind cluster", script: 'kind create cluster')
+        sh(label: "Create kind cluster", script: 'kind create cluster --config - $PWD/../../testing/environments/kubernetes/kind-config.yaml')
```
[the remark this answers is not in the document]
Comment (ChrsMark) [reply .ci/Jenkinsfile]: ah ok! I can move it there!

**Review by mtojek — APPROVED** [review]
If the CI is happy, please consider this PR as approved :)

File: .ci/Jenkinsfile [hunk 2]
```diff
@@ -185,7 +185,7 @@ def withKubernetes(Closure body) {
           chmod +x ${HOME}/bin/kubectl
           kubectl version --client
           ''') }
-        sh(label: "Create kind cluster", script: 'kind create cluster')
+        sh(label: "Create kind cluster", script: 'kind create cluster --config ../../kind-config.yaml')
```
Comment (jsoriano) [inline .ci/Jenkinsfile:188]: If this config is in the root of the project, try with `BASE_DIR`

```suggestion
        sh(label: "Create kind cluster", script: 'kind create cluster --config ${BASE_DIR}/kind-config.yaml')
```

Comment (ChrsMark) [reply .ci/Jenkinsfile:188]: it seems to pass with `../../kind-config.yaml` 😅

**Pull request merged**